Packs/FireEyeETP/Integrations/FireEyeETPEventCollector/FireEyeETPEventCollector.py

```diff
@@ -220,6 +220,11 @@ def fetch_command(self, demisto_last_run: dict, first_fetch: None | datetime = N
                 demisto.debug(f"{LOG_LINE} getting events of type {event_type.name}")
                 if event_type.client_max_fetch > 0:
                     next_run, new_events = self.get_events(event_type=event_type, last_run=next_run)
+                    # annotate direction for non-activity events
+                    if event_type.name != "activity_log":
+                        direction = "outbound" if event_type in OUTBOUND_EVENT_TYPES else "inbound"
+                        for evt in new_events:
+                            evt.setdefault("direction_source", direction)
```

Comment:
Its probably affect less the performance if we do it in the fetch_email_trace and in the fetch_alerts functions with lambda.